src/components/TagsDisplay.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react'
+import './TagsDisplay.css'
+
+interface Props extends React.HTMLProps<HTMLDivElement> {
+    tags: string[]
+}
+
+class component extends React.Component<Props> {
+    render() {
+        const { tags } = this.props
+        return (
+            <div className={`cl-tags-display ${this.props.className}`}>
+                {tags.length === 0
+                    ? `0 tags`
```

Comment:
Perhaps "No Tags" is more user friendly?

Reply:
Yea, changed. I see what people mean by being confused about GitHub not showing the updated file on the review comment. Here it is: https://github.com/Microsoft/ConversationLearner-UI/pull/989/files#diff-c19a8069a2276942e428e7b1b27f112bR14

I think there actually are two different cases for the text and context is only known by the parent. I didn't implement it yet but I will if you think it's worth it.

The `<TagsReadOnly />` component my be used in various places and when there are no tags given it needs to still render something useful like `No Tags` to the user. However, on the `DetailsList` then I think it should say something similar to the "description" column like: "Open dialog to add tags"

Reply:
I added it in latest push